tools/cli/flags.go

```diff
@@ -287,6 +288,10 @@ var flagsForExecution = []cli.Flag{
 		Name:  FlagRunIDWithAlias,
 		Usage: "RunID",
 	},
+	cli.StringFlag{
+		Name:  FlagJWT,
+		Usage: "JWT token for authentication",
```

Comment:
this is authorization not authentication :D 

We have a different story for authentication: https://github.com/uber/cadence/issues/4095